src/main.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+
+import { NFC } from 'nfc-pcsc';
+import EthTx from 'ethereumjs-tx';
+import LuksoNfcChip from './reader/LuksoNfcChip';
+import Tlv from './tlv/Tlv';
```

Comment:
What is the tlv?

Reply:
TLV is a Tag(Type)-Length-Value object. We use TLV to encode DATA field of APDU command and response.